src/js/utils.js

```diff
@@ -108,8 +108,7 @@ function explodeSubdomains(fqdn, all) {
 /*
  * Estimate the max possible entropy of str using min and max
  * char codes observed in the string.
- * Tends to overestimate in many cases, e.g. hexadecimals.
- * Also, sensitive to case, e.g. bad1dea is different than BAD1DEA
+ * Sensitive to case, e.g. bad1dea is different than BAD1DEA
  */
 function estimateMaxEntropy(str) {
```

Comment:
Are these changes beneficial for the local storage ("supercookie") case as well?

Reply:
Still a question.

Reply:
Yes, they should be. These changes are all meant to tighten up the way the heuristic worked before -- entropy = (size of character set) * (number of characters).

Reply:
Should add unit tests to exercise `estimateMaxEntropy` code paths.